pkg/domain/entities/manifest.go

```diff
@@ -43,6 +43,8 @@ type ManifestAddOptions struct {
 type ManifestAnnotateOptions struct {
 	// Annotation to add to manifest list
 	Annotation []string `json:"annotation" schema:"annotation"`
+	// Annotation to add to manifest list by a map
```

Comment:
Can you add that `annotations` is preferred over `annotation` to the comment?

Reply:
Done.